k2/csrc/fsa_algo.cc

```diff
@@ -150,6 +153,148 @@ void Connect(const Fsa &a, Fsa *b, std::vector<int32_t> *arc_map /*=nullptr*/) {
   }
 }
 
+inline int32_t InsertIntersectionState(
```

Comment:
It would be better if this function is placed inside the above unnamed namespace.

Reply:
Sure